recipe/meta.yaml

```diff
@@ -19,19 +19,20 @@ build:
 
 requirements:
   build:
-    - pkg-config  # [not win]
+    - pkg-config =0.29.2  # [not win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
     - python
-    - cython >=0.29
     - packaging
     - pip
     - setuptools
     - wheel
-    - zeromq
+    - libsodium =1.0.18
+    - zeromq =4.3.4
   run:
     - python
+    - libsodium
```

Comment:
If we pin a package in `host` then pin a supported range in `run` like **libsodium >=1.0.19,<1.0.19.0a0**.
Here is `run_exports` of this package https://github.com/AnacondaRecipes/libsodium-feedstock/blob/63f7a518a4e8b22cf223604d61017903e0cb36ad/recipe/meta.yaml#L17.
That's needed because `conda-build` has a bug in this kind of use case.